addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AddOnJobDialog.java

```diff
@@ -58,7 +58,8 @@ public AddOnJobDialog(AddOnJob job) {
 
         this.addTextField(NAME_PARAM, this.job.getData().getName());
 
-        this.addCheckBoxField(UPDATE_ADDONS_PARAM, this.job.getParameters().getUpdateAddOns());
+        // Disabled until this can work without breaking things
```

Comment:
Could you add a task tag? (e.g. `XXX` or `FIXME`)

Reply:
Done :)